http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/InstrumentationEnabledConfig.scala

```diff
@@ -0,0 +1,5 @@
+package org.broadinstitute.dsde.workbench.leonardo.config
+
+case class InstrumentationEnabledConfig(
```

Comment:
I don't think we need this to be a config on it's own....it should just be part of `AKSInterpreterConfig`